models.py

```diff
@@ -249,6 +249,7 @@ def FetchAllEntities(cls):
 class Dayparts(ndb.Model):
   """Examples: Monday AM, or M-W-F 8am-9am"""
   data = ndb.TextProperty()
+  jdata = ndb.JsonProperty()
```

Comment:
I didn't know enough to be surprised by this. I was really surprised the rest of the site just worked with JsonProperty instead of TextProperty.

The benefit of storing data and jdata in the same place is that the code for the Setup pages didn't need to be changed. You can access the old yaml data as before using Fetch. When you want Json, use FetchJson.